python_modules/dagit/dagit/webapp/src/SidebarTabbedContainer.tsx

```diff
@@ -43,6 +42,22 @@ const TabInfo: Array<ITabInfo> = [
 export default class SidebarTabbedContainer extends React.Component<
   ISidebarTabbedContainerProps
 > {
+  static fragments = {
+    SidebarTabbedContainerPipelineFragment: gql`
+      fragment SidebarTabbedContainerPipelineFragment on Pipeline {
+        ...SidebarPipelineInfoFragment
+      }
+
+      ${SidebarPipelineInfo.fragments.SidebarPipelineInfoFragment}
+    `,
+    SidebarTabbedContainerSolidFragment: gql`
+      fragment SidebarTabbedContainerSolidFragment on Solid {
+        ...SidebarSolidInfoFragment
+      }
+      ${SidebarSolidInfo.fragments.SidebarSolidInfoFragment}
+    `
+  };
+
```

Comment:
👍 Definitely wish something could automatically roll these up the component tree, but I see the pattern now. Thanks for apply these changes!